components/header.js

```diff
@@ -1,7 +1,39 @@
-import React from 'react';
+"use client";
```

Comment:
can this be in single quotes to match the rest of the imports up here? idk if your linter will yell at your or not

```suggestion
'use client';
```